### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by eneufeld**

### CODE REVIEW

File: web/README.md [hunk 1]
```diff
@@ -25,25 +25,18 @@ Install yarn.
...
     git clone https://github.com/eclipsesource/coffee-editor.git
     cd coffee-editor
-    yarn install
-    cd browser-app
-    yarn start
+    ./run.sh -b -c -f
+    
+    git clone git@github.com:eclipsesource/graphical-lsp.git
+    cd graphical-lsp
+    Open in Eclipse and start the com.eclipsesource.glsp.example.workflow.ExampleServerLauncher Java class
```
Comment (planger) [inline web/README.md]: I think this line shouldn't be indented, but rather a normal paragraph.

**Comment by tortmayr** [comment]
I don't think it's a good idea to directly clone and use the GLSP repository here. This might lead to issues if the version of the Workflow example server becomes incompatible to the currently used versions of `@glsp-examples/workflow-theia`. (Even if the depndency is set to "next" the version is soft pinned in the yarn.lock file)
It would be e better to use a checked-in fatjar for the Workflow GLSP server. This way the server can be started via run script as well.

**Comment by eneufeld** [comment]
I agree, but that would be a new pr then, right?

**Comment by planger** [comment]
> I agree, but that would be a new pr then, right?

Yes, I think that can be done in an own PR. Thanks!

**Pull request merged**